genome_kit/vcf_table.py

```diff
@@ -517,13 +517,17 @@ def _masked_results(self, results):
 
     # values for GT (as per cyvcf2)
     GT_HOMOZYGOUS_REF = 0
-    """Constant indicating a ``0/0`` in GT format value."""
-    GT_HETEROZYGOUS = 1
+    """Constant indicating a ``0/0`` or ``0|0`` in GT format value."""
+    GT_HETEROZYGOUS_UNPHASED = 1
     """Constant indicating a ``0/1`` in GT format value."""
     GT_HOMOZYGOUS_ALT = 2
-    """Constant indicating a ``1/1`` in GT format value."""
+    """Constant indicating a ``1/1`` or ``1|1`` in GT format value."""
     GT_UNKNOWN = 3
-    """Constant indicating a ``?/?`` in GT format value."""
+    """Constant indicating a ``?/?`` or ``?|?`` in GT format value."""
+    GT_HETEROZYGOUS_PHASED_0_1 = 4
+    """Constant indicating a ``0|1`` in GT format value."""
+    GT_HETEROZYGOUS_PHASED_1_0 = 5
+    """Constant indicating a ``1|0`` in GT format value."""
```

Comment:
can the constants be changed to np.int8(...), to reduce the filesize?

Sorry this was my bug from way back.